### ISSUE TITLE
Evaluation UI should warn me when classes not in training set are present in test set

### ISSUE THREAD

**Issue by srikris** [apple/turicreate#1937]
When I evaluate the model and use the evaluation UI, the classes not in the training set that are in the test set should have a warning 

Currently, this is what I get

<img width="1171" alt="image" src="https://user-images.githubusercontent.com/1875987/58290138-7fc47700-7d6d-11e9-8eb0-412d2ca59f47.png">
